azure-amqp/cpp/main.cpp

```diff
@@ -33,31 +33,24 @@
 #include "iothub_message.h"
 #include "iothubtransportamqp.h"
 
+// define the following if not using a board with sensors
+//#define SIMULATE_DEVICES
+
+// Define the following if using a Grove Pi Shield for UP2 board
+#define USING_GROVE_PI_SHIELD
+
 #include <iostream>
 
 #include "certs.h"
 #include "credentials.h"
 #include <mraa.hpp>
-#include <grove.hpp>
 
-/**
- * This project template allow you to test the Microsoft* Azure* IoT Hub.
- * It will read the temperature from the Grove Temperature Sensor and send
- * it to the Azure* IoT Hub.
- *
- * First steps:
- * - Create an IoT HuB following the instruction in
- * https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal
- * - Create a connection string for a device following the instruction in
- * https://aka.ms/manageiothub
- * - Copy the connection string inside the "credentials.h" header
- * - Run the example and check the results inside your Azure* IoT Hub
- *
- * * Other names and brands may be claimed as the property of others.
- */
```

Comment:
Tom's usability study observation recommend that we put link to the README.md file here : 
"For instructions on how to setup Azure IoT Hub, go to <README.md link>"